experimental/dds/tree2/src/feature-libraries/sequence-field/compose.ts

```diff
@@ -940,15 +609,68 @@ function getIntention(
 	return rev === undefined ? undefined : revisionMetadata.getInfo(rev).rollbackOf ?? rev;
 }
 
-function areInverseRevisions(
-	rev1: RevisionTag | undefined,
-	rev2: RevisionTag | undefined,
-	revisionMetadata: RevisionMetadataSource,
-): boolean {
-	if (rev1 === undefined || rev2 === undefined) {
-		return false;
+// TODO: Try to share more logic with the version in rebase.ts.
+function compareCellPositions(
+	baseCellId: DetachEvent,
+	baseMark: ExistingCellMark<unknown>,
+	newMark: EmptyInputCellMark<unknown>,
+	newRevision: RevisionTag | undefined,
+	cancelledInserts: Set<RevisionTag>,
+	gapTracker: GapTracker,
+): number {
```

Comment:
TODO: document return value:
0: cells start at the same place.
-n: there are n cells in base before first cell in new
n: there are n cells in new before first cell in base
(n can be inifinite when all the cells in...)

Reply:
Done